pkg/ts/catalog/chart_catalog.go

```diff
@@ -4089,8 +4089,9 @@ var charts = []sectionDescription{
 			{
 				Title: "Total RU consumption",
 				Metrics: []string{
-					"tenant.consumption.request_units",
+					"tenant.consumption.cross_region_network_ru",
 					"tenant.consumption.kv_request_units",
+					"tenant.consumption.request_units",
```

Comment:
Will the order affect anything here in a backport?

Reply:
he chart_catalog is used by the DB console UI on the host cluster. Since these metrics are only used internally by Serverless, the order doesn't matter.